src/NServiceBus.NHibernate/NServiceBus.NHibernate.csproj

```diff
@@ -1,30 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
I wonder if we need to specifically document that although NSB supports net452, the NHibernate dependency forces at least 461?

Reply:
Isn't nuget range specification sufficient enough?